frontend/src/components/admin/admin.tsx

```diff
@@ -23,18 +36,27 @@ const Admin: React.FC = () => (
         <Col sm={8} lg={8} md={8}>
           <Nav variant="tabs" defaultActiveKey="/">
             <Nav.Item>
-              <Nav.Link href="/">Your Company</Nav.Link>
+              <Link to={AppRoute.ADMIN}>
```

Comment:
If you use html you can avoid some custom styles. 
Use html elements with bootstrap classes